src/ui/public/vis/components/tooltip/__tests__/positioning.js

```diff
@@ -147,6 +147,31 @@ describe('Tooltip Positioning', function () {
         }
       });
     });
+
+    it('ident only west overflow when tooltip overflows both sides of inner container but outer contains tooltip', function () {
```

Comment:
I think I understand the logic but this test makes me a little confused.  If the tooltip is positioned in the center, and overflows both sides of the inner container evenly, and the inner container is centered within the outer, is this test just checking that west is the default in the case of a tie?

I think what might make it a little clearer is if there were two tests.  Same set up where the tooltip overflows both sides of the inner chart, but have one test where the outer window has extra space on the left, and ensure it picks the right as overflow, and another where the outer window has extra space on the right, and ensure it picks the left as overflow.  

Might also be interesting to add a test where the tooltip doesn't overflow both sides of the inner container evenly, but for instance, it has less overflow on the right, but the outer window would cut it off on the left, and could handle it on the right, so it still pushes the tooltip to the right.

A few extra tests just might also help readers understand the logic because it took me awhile to figure out what was going on.

Also just tiny nit, I would write out identifies, at first I thought it might have been a misspelling of indent and had trouble reading the description.